Userland/Utilities/netstat.cpp

```diff
@@ -316,6 +331,9 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
                 columns[peer_address_column].buffer = get_formatted_address(peer_address, peer_port);
             if (state_column != -1)
                 columns[state_column].buffer = "-";
+            if (flag_extend && user_column != -1) {
+                columns[user_column].buffer = get_formatted_user(origin_uid);
+            }
```

Comment:
You can leave out these braces for single-line if bodies

Reply:
Yep, there where multiple lines at some point. Forgot to drop them.